cuoiky:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  tt_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: tt_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}

  tp_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: tp_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}

  sp_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: sp_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}

  st_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: st_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}

  xoay_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: xoay_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}

  tien_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: tien_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
